daemon/api.go

```diff
@@ -55,7 +55,7 @@ import (
 	"github.com/snapcore/snapd/store"
 )
 
-var api = []*Command{
+var privateAPI = []*Command{
```

Comment:
To be honest I'm not sure how to do that. From what I can see, the only way to differentiate between the two sockets is at the listener level. Once we get to the request level (e.g. `canAccess()`, where the `UserOK` etc. is checked) I don't see a way to trace back to the listener from which the request originated. I'm probably missing something obvious.